client/src/app/components/dialogs/delete-task/delete-task.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="title">
```

Comment:
I guess that we should have like common dialog component which we can use in many places, it should not be only for deletion tasks